coordinator-common/src/main/java/org/apache/kafka/coordinator/common/CoordinatorRecord.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.coordinator.group;
+package org.apache.kafka.coordinator.common;
```

Comment:
I wonder if we should move `CoordinatorRecord`, `CoordinatorMetrics`, `CoordinatorMetricsShard` and `CoordinatorRuntimeMetrics` to the `runtime` package too. They are all generic classes which are part of the runtime framework in my opinion. What do you think?